ontolib-core/src/main/java/com/github/phenomics/ontolib/ontology/data/ImmutableOntology.java

```diff
@@ -193,9 +190,33 @@ public TermId getRootTermId() {
     final Set<TermId> childTermIds = OntologyTerms.childrenOf(subOntologyRoot, this);
     final ImmutableDirectedGraph<TermId, ImmutableEdge<TermId>> subGraph =
         (ImmutableDirectedGraph<TermId, ImmutableEdge<TermId>>) graph.subGraph(childTermIds);
-    return new ImmutableOntology<T, R>(metaInfo, subGraph, subOntologyRoot,
-        Sets.intersection(nonObsoleteTermIds, childTermIds),
-        Sets.intersection(obsoleteTermIds, childTermIds), termMap, relationMap);
+    Set<TermId> intersectingTerms = Sets.intersection(nonObsoleteTermIds,childTermIds);
```

Comment:
Whitespace is inconsistent. Your IDE should be able to help you with auto-formatting.